src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1209,6 +1240,8 @@
           text: >
             az network nic create -g MyResourceGroup --vnet-name MyVnet --subnet MySubnet -n MyNic \\
                 --ip-forwarding --network-security-group MyNsg
+        - name: Create a network interface for a specified subnet on a virtual network with network security group and application security groups.
+          text: az network nic create -g MyResourceGroup --vnet-name MyVnet --subnet MySubnet -n MyNic --network-security-group MyNsg --network-security-group MyNsg --application-security-groups Web, App
```

Comment:
Long examples should flow with the `|` operator. Include `\\` continuations to break across multiple lines in the shell.